Adding bloom command meta data, bloom group and bloom data type documentaion (#233)
valkey-io/valkey-doc: commands/bf.reserve.md

Creates an empty bloom object with the capacity and false positive rate specified

## Reserve fields

> * error_rate - The false positive rate the bloom filter will be created with

Reviewer: ```suggestion
* error_rate - The false positive rate of the bloom filter
```

> Creates an empty bloom filter with the capacity and false positive rate specified. By default, a scaling filter is created with the default expansion rate.

Reviewer: ```suggestion
Creates an empty bloom filter with the specified capacity and false positive rate. By default, a scaling filter is created with the default expansion rate.
```
